### ISSUE TITLE
Should I use bias in convolution ops?

### ISSUE THREAD

**Issue by KeanuWongves** [CQFIO/FastImageProcessing#7]
I realized that the slim.con2d，if you use the normalizer_fn , it says the conv will have no bias.
but there comes the question since it uses adaptive BN ，as I can see in the code the initial parameter is w0=1 and w1=0, which means at first it doesn't use batch normalization, So the bias is used in futher forward-propagation.and even though w1 is not 0, as long as the w0 do not becoming to 0, I think the bias is still used in someway.So，Should I use bias in conv？I was doing this in pytorch.it doesn't have a function like slim.conv2d.
